tests/conftest.py

```diff
@@ -18,21 +18,25 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import pytest
+from flask import Flask
 
-import unittest
+from lsst.dax.imgserv import api_soda as is_api_soda
 
-class DALUnitTest(unittest.TestCase):
-    """Unit tests for DAL service components.
 
-    TODO: Implement these tests for generic SODA.
-    """
+@pytest.fixture
+def app():
+    app = Flask(__name__)
+    app.register_blueprint(is_api_soda.image_soda, url_prefix='/api/image/soda')
```

Comment:
The URL that this is running at will likely need to be configurable in the future, but I guess this is fine for now

Reply:
Yeah.